src/node_crypto.cc

```diff
@@ -3185,7 +3185,7 @@ static ManagedEVPPKey GetPublicOrPrivateKeyFromJs(
           is_public = false;
           break;
         default:
-          CHECK(!"Invalid key encoding type");
```

Comment:
Using a string here makes the error a little more informative:
```console
Assertion `!"Invalid key encoding type"' failed.

 1: 0x100063171 node::Abort() [nodejs/node/./node]
 2: 0x1000630ae node::PrintErrorString(char const*, ...) [/nodejs/node/./node]
```
Compared to:
```console
Assertion `false' failed.

 1: 0x100063171 node::Abort() [/nodejs/node/./node]
 2: 0x1000630ae node::PrintErrorString(char const*, ...) [/nodejs/node/./node]
```

Reply:
Thanks! Definitely agree that the string makes it more informative.
The main intention for changing this was because of the clang `-Wstring-conversion` warning.
Also, it's easy to forget the `!` in front of the string.
I like @bnoordhuis 's idea, pushing a change for that now.